conda_smithy/lint_recipe.py

```diff
@@ -91,13 +91,9 @@ def get_list_section(parent, name, lints, allow_single=False):
 
 
 def lint_section_order(major_sections, lints):
-    section_order_sorted = sorted(
-        major_sections, key=EXPECTED_SECTION_ORDER.index
-    )
+    section_order_sorted = sorted(major_sections, key=EXPECTED_SECTION_ORDER.index)
```

Comment:
This was changed by black and now it's reverted? I don't understand

Reply:
I don't understand either maybe @mariusvniekerk  has his black setup to use longer / shorter line lengths?